scss/detail.scss

```diff
@@ -56,31 +47,41 @@ $badge-width-responsive: calc(#{$badge-width} * 0.7);
     content: "";
     display: block;
     height: $badge-height;
-    left: 194px;
+    left: 192px;
```

Comment:
why this change?

Reply:
Nothing specific just wanted to align it more.